pkg/api/entries.go

```diff
@@ -299,6 +299,9 @@ func getEntryURL(locationURL url.URL, uuid string) strfmt.URI {
 func GetLogEntryByUUIDHandler(params entries.GetLogEntryByUUIDParams) middleware.Responder {
 	logEntry, err := retrieveLogEntry(params.HTTPRequest.Context(), params.EntryUUID)
 	if err != nil {
+		if errors.Is(err, ErrNotFound) {
+			return handleRekorAPIError(params, http.StatusNotFound, err, "")
+		}
 		if _, ok := (err).(types.ValidationError); ok {
 			return handleRekorAPIError(params, http.StatusBadRequest, err, "incorrectly formatted uuid %s", params.EntryUUID)
```

Comment:
I think this will also need a `fmt.Sprintf("incorrect formatted uuid %s", params.EntryUUID)`, otherwise the error message includes the `%s`.

(I couldn't suggest a change since this line didn't change, thanks GitHub! 🤬)